tools/tasks/seed/build.js.dev.ts

```diff
@@ -31,7 +31,7 @@ export = () => {
     .pipe(plugins.typescript(tsProject));
 
   return result.js
-    .pipe(plugins.sourcemaps.write())
+    .pipe(plugins.sourcemaps.write('.', {includeContent: false, sourceRoot: '../../src/client'}))
```

Comment:
No effect beside generate the source map (.map.js) for each .js in the dist/dev folder.
The command `npm start` (and everything else) works as expected.
Feel free to clone the PR and try, but i have already tested.
The only difference is for tooling who use sourcemaps, but i think that this way is even more compatible since the sourceRoot is set according to the path.

I even think that this change should go to the parent seed.
But i did on this seed because of the number of tasks to test and debug in different platforms (web, electron, nativescript).